packages/tablev2/style.ts

```diff
@@ -79,8 +79,11 @@ export const sortable = (dir: "asc" | "desc" | null) => css`
       "currentColor"
     )};
   }
+`;
+
+export const headerHover = css`
   &:hover {
-    &:after {
+    #sortIcon: after {
```

Comment:
Could we use a classname here instead of an id and retain this functionality?

Should be able to combine classNames with Emotion similar to this:
`className={cx(style.sortable(order), sortIcon)}`

Reply:
@nataliepina updated please check

Reply:
yes, @nataliepina have updated them. What are your thoughts on snapshot testing? Changing a simple CSS is causing snapshots to fail.

Reply:
I get you. We have put this on our agenda to discuss what we want to do moving forward. So for now we will keep the snapshots.